makefile

```diff
@@ -14,14 +16,18 @@ endef
 
 # CI build
 .PHONY: ci
-ci: gen-java gen-swagger
+ci: gen-go gen-java gen-swagger
 
 # Generate ProtoBuf implementation for Go.
 .PHONY: gen-go
 gen-go:
-	$(foreach file,$(PROTO_FILES),$(call exec-command,protoc --go_out=plugins=grpc:$(GOPATH)/src $(file)))
 	rm -rf ./$(GENDIR)/go
-	cp -R $(GOPATH)/src/github.com/open-telemetry/opentelemetry-proto/$(GENDIR)/go ./gen/
```

Comment:
Change 2: This line fails if the source is checked out into `$(GOPATH)/src/github.com/open-telemetry/opentelemetry-proto`, which is actually a canonical path for Go, and it's being used by Travis CI.

So I made this `cp` conditional: we only need to copy generated files if we are in the custom directory.